app/src/test/java/org/mozilla/fenix/tabtray/TabTrayFragmentInteractorTest.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabtray
+
+import io.mockk.mockk
+import org.junit.Test
+import io.mockk.verify
+
+class TabTrayFragmentInteractorTest {
+    private val controller = mockk<TabTrayController>(relaxed = true)
+    private val interactor = TabTrayFragmentInteractor(controller)
+
+    @Test
+    fun onNewTabTapped() {
+        interactor.onNewTabTapped(private = true)
+        verify { controller.onNewTabTapped(true) }
+
+        interactor.onNewTabTapped(private = false)
+        verify { controller.onNewTabTapped(false) }
+    }
+
+    @Test
+    fun onTabTrayDismissed() {
+        interactor.onTabTrayDismissed()
+
```

Comment:
Should we remove these new line to be consistent with the formatting in `onNewTabTapped` between the `interactor` and `verify`?